src/test/scala/lenthall/util/TryUtilSpec.scala

```diff
@@ -0,0 +1,57 @@
+package lenthall.util
+
+import lenthall.exception.AggregatedException
+import lenthall.util.TryUtil._
+import org.scalatest.{FlatSpec, Matchers}
+
+import scala.compat.Platform.EOL
+import scala.util.{Failure, Success}
+
+class TryUtilSpec extends FlatSpec with Matchers {
+
+  behavior of "TryUtil"
+
+  it should "stringify successes" in {
```

Comment:
This test looks like it's not stringifying successes since the result is an empty traversable? Possibly rename this test?